models.py

```diff
@@ -11,10 +10,11 @@ class ToDo(Base):
     id: Mapped[int] = mapped_column(Integer, primary_key=True, index=True)
     content: Mapped[str] = mapped_column(String)
     session_key: Mapped[str] = mapped_column(String)
+    notes: Mapped[str | None] = mapped_column(Text, nullable=True)  # New column added
 
 
-def create_todo(db: Session, content: str, session_key: str):
-    todo = ToDo(content=content, session_key=session_key)
+def create_todo(db: Session, content: str, session_key: str, notes: str = None):
+    todo = ToDo(content=content, session_key=session_key, notes=notes)
     db.add(todo)
```

Comment:
The `create_todo` function now includes the `notes` parameter. Ensure all calling code is updated to handle this new optional parameter.